TelemetrySite/server/endpoints/flask_endP.py

```diff
@@ -0,0 +1,105 @@
+from flask import Flask, jsonify, request
+from flask_restful import reqparse
+import utils
+
+
+app = Flask(__name__)
+
+# Get specific BmsConfig record
+@app.route('/bmsconfig/<int:id>', methods=['GET'])
+def get_bmsconfig(self):
+    try:
+        buttonID = request.args.get('buttonID', default= -1)
+
+        if(int(buttonID) >= 1):
+            print("DING")
+            
+        sql = "SELECT * FROM BmsConfig WHERE id = %s"
```

Comment:
Instead of using `SELECT *`, it would be better to list out the tables explicitly, so this script won't break if the table orders change